packages/html-to-slate-ast/package.json

```diff
@@ -9,15 +9,16 @@
     "test": "tsdx test --passWithNoTests",
     "test:watch": "tsdx test --watch --passWithNoTests",
     "lint": "tsdx lint",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "test:node": "node examples/node-script.js"
   },
   "peerDependencies": {
     "slate": "^0.66.1",
     "slate-hyperscript": "^0.67.0"
   },
   "devDependencies": {
-    "@types/jsdom": "^21.1.5",
-    "jsdom": "^22.1.0",
+    "@types/jsdom": "^21.1.6",
+    "jsdom": "^24.0.0",
```

Comment:
if jsdom is required on runtime, and has to be explicitly installed by the user, shouldn't it be included in `peerDependencies` ?

Reply:
I'm not sure, as the package can also be used on a browser environment, and in this case, jsdom is not required.

Reply:
you can also mark peer dependencies as optional: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta

Reply:
Updated on [8450225](https://github.com/hygraph/rich-text/pull/121/commits/84502258f5a0bfbe7b7361ecb4dccce1058cdfce)